honeybadgermpc/protocols/avid.py

```diff
@@ -108,7 +108,7 @@ def broadcast(self, o):
 
                     return decoded_output
 
-    async def disperse(self, sid, pid, input_list):
+    async def disperse(self, sid, pid, input_list, client_mode=False):
```

Comment:
Update the docstring to say what a client mode is.

Basically in the client mode, the dealer just sends the first message to all the receivers and returns.